common/ts/prepare-task.ts

```diff
@@ -12,7 +12,7 @@ export default async function prepareTask(endpoint: Endpoint, rootPath: string)
 
   const props: { [key: string]: string } = {};
 
-  if (endpoint.type === EndpointType.SonarCloud) {
+  if (endpoint.type === EndpointType.SonarCloud || branchFeatureSupported(endpoint)) {
```

Comment:
<nitpicking>for the sake of symmetry I would also extract the sonarcloud check in a small method</nitpicking>